### PR TITLE
added my profile

### PR DESCRIPTION
**Pull request by DEmosus**
I've just added my profile.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2891,13 +2891,13 @@
             }
```
Comment (sophiabrandt) [inline package-lock.json:2891]: Please remove the changes to the `package-lock.json` file

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (DEmosus) [reply package-lock.json:2891]: Does this mean I need to redo the whole process?

**Review by DEmosus — COMMENTED** [review]
update the profile image also

File: package-lock.json (hunk 1 above)
Comment (DEmosus) [reply package-lock.json]: I've removed the changes. 
Thanks.

**Pull request merged**